src/settingsview.cpp

```diff
@@ -30,6 +34,7 @@ SettingsView::SettingsView(QWidget *parent)
     ui->browseButton->setText(gt("browse"));
     ui->monitorClear->setText(gt("clear"));
     ui->monitorBrowse->setText(gt("browse"));
+    ui->downloadDirPathCopy->setText("<img src=':/resources/icons/copy.svg'>");
```

Comment:
This doesn't seem to use the `copy.svg` icon included in this PR:

![Screenshot from 2024-06-16 15-06-48](https://github.com/kiwix/kiwix-desktop/assets/11933502/41bf7b75-1c2b-47de-863a-fa310afe13e8)

Reply:
For some reason I just assumed this was render issue on my side, but of course it is not. Just used the wrong path relative to the .qrc file